2020/28xxx/CVE-2021-28648.json

```diff
@@ -0,0 +1,63 @@
+{
```

Comment:
Expected location (2021/28xxx/CVE-2021-28648.json) of file does not match actual location (2020/28xxx/CVE-2021-28648.json)